### ISSUE TITLE
Only 5 sites I had trouble with

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MiraculousLegendas/get-flash-videos#4]
```
Great product, thank you.  I am running 1.1 on CentOS 5.3.

I had trouble with the following sites:

1)
get_flash_videos
http://www.milkeninstitute.org/events/gcprogram.taf?function=detail&EvID=1630&ev
entid=GC09
[1] 23914
[2] 23915
Downloading http://www.milkeninstitute.org/events/gcprogram.taf?function=detail
Using method 'generic' for
http://www.milkeninstitute.org/events/gcprogram.taf?function=detail
Guessed http://www.milkeninstitute.org/events/multimedia/playlistmain.xml
trying...
Error: No URLs found at /home/user/bin/get_flash_videos line 1987.
Couldn't extract Flash movie URL. This site may need specific support adding,
or fixing.

2)
get_flash_videos http://www.simoleonsense.com/video-focus-and-experiences/
Downloading http://www.simoleonsense.com/video-focus-and-experiences/
Using method 'generic' for
http://www.simoleonsense.com/video-focus-and-experiences/
Guessed http://www.simoleonsense.com/video-focus-and-experiences/Training
trying...
Error: No URLs found at /home/user/bin/get_flash_videos line 1987.

3)
get_flash_videos http://learnbythedrop.com/drop/144
Downloading http://learnbythedrop.com/drop/144
Using method 'google' for http://learnbythedrop.com/drop/144
Error:   
Couldn't extract Flash movie URL. This site may need specific support adding,
or fixing.

4)
get_flash_videos
http://www.simoleonsense.com/a-venture-capitalists-speaks-at-google-on-technolog
ical-disruption/
Downloading
http://www.simoleonsense.com/a-venture-capitalists-speaks-at-google-on-technolog
ical-disruption/
Using method 'generic' for
http://www.simoleonsense.com/a-venture-capitalists-speaks-at-google-on-technolog
ical-disruption/
Error: No URLs found at /home/user/bin/get_flash_videos line 1987.
Couldn't extract Flash movie URL. This site may need specific support adding,
or fixing.

5)
get_flash_videos
http://optionarmageddon.ml-implode.com/2009/05/14/a-longitudinal-study-on-happin
ess/
Downloading
http://optionarmageddon.ml-implode.com/2009/05/14/a-longitudinal-study-on-happin
ess/
Using method 'brightcove' for
http://optionarmageddon.ml-implode.com/2009/05/14/a-longitudinal-study-on-happin
ess/
Must have Data::AMF::Packet installed to download Brightcove videos 
[Note: I have this
http://rpmfind.net/linux/RPM/dag/redhat/el5/x86_64/perl-Data-AMF-0.02004-1.el5.r
f.noarch.html
installed, which seems to include Packet.]

Thanks again, great product!
```

Original issue reported on code.google.com by `somebody...@gmail.com` on 16 May 2009 at 11:20

**Comment by GoogleCodeExporter** [comment]
```
1) You need to quote the URL as it has & in. Although that page doesn't work as 
it
uses RTMP underneath and generic doesn't support extracting RTMP urls (yet..).
However if you use the URL in the link below it ("Click here if you are having 
video
playback problems") it works fine.

2) I've added support for fora.tv

3) Seems to work for me.. (maybe try the development version).

4) That seems to link to slideshare rather than a video..

5) That works for me, are you sure you have all the dependencies installed?
e.g. try running:
  perl -MData::AMF::Packet -e1
and see if you get an error.
```

Original comment by `monsieur...@gmail.com` on 17 May 2009 at 10:35

**Comment by GoogleCodeExporter** [comment]
```
Okay, thanks.
3) I could not get the development version to work.  I'll just wait for the 
next release.
5) My bad, I installed missing dependencies shown by your command.  I get errors
still (Download failed, no valid file downloaded), but I suspect these are also 
fixed
in dev version.

```

Original comment by `somebody...@gmail.com` on 17 May 2009 at 1:17

**Comment by GoogleCodeExporter** [comment]
```
Should be resolved by 1.12.
```

Original comment by `monsieur...@gmail.com` on 18 May 2009 at 8:01
- Changed state: **Verified**
